### PR TITLE
Update working-at-mattermost.rst

### PR DESCRIPTION
**Pull request by comharris**
Updated USA/Canada holidays for 2020

@lindy65 FYI (you may have already done this, but adding just in case)

### CODE REVIEW

**Comment by lindy65** [comment]
Thanks @comharris - I hadn't updated 2020 holidays ...

File: source/process/working-at-mattermost.rst [hunk 1]
```diff
@@ -118,27 +118,30 @@ Please take off holidays relevant to your culture, region/country and preference
...
+
+- New Year's Day (1/1/2020)
+- US MLK Jr. Day (1/20/2020)
+- Canadian Family Day and US Presidents' Day (2/17/2020)
+- US and Canadian Good Friday (4/10/2020)
+- Canadian Victoria Day (5/18/2020)
+- US Memorial Day (5/25/2020)
+- Canada Day (7/1/2020)
+- Canada Day additional statutory holiday (7/2/2020)
+- US Independence Day (7/4/2020*)
```
Comment (amyblais) [inline source/process/working-at-mattermost.rst]: Using asterisks will make the rest of the text either cursive or bold, so we'll have to think about another way to add the references.

Comment (comharris) [reply source/process/working-at-mattermost.rst]: Changed to `+` - thanks @amyblais

Comment (amyblais) [inline source/process/working-at-mattermost.rst]:
```suggestion
- US Independence Day (7/4/2020+)
```

File: source/process/working-at-mattermost.rst [hunk 2]
```diff
@@ -118,27 +118,30 @@ Please take off holidays relevant to your culture, region/country and preference
...
+- Canadian Civic / Provincial Day (8/3/2020)
+- US Labor Day, Canadian Labour Day (9/7/2020)
+- Canadian Thanksgiving (10/12/2020)
+- US Thanksgiving (11/26/2020)
+- US and Canadian Christmas Day (12/25/2020)
+- Canadian Boxing Day (12/26/2020+)
```
Comment (amyblais) [inline source/process/working-at-mattermost.rst]:
```suggestion
- Canadian Boxing Day (12/26/2020++)
```

File: source/process/working-at-mattermost.rst [hunk 3]
```diff
@@ -118,27 +118,30 @@ Please take off holidays relevant to your culture, region/country and preference
...
 
 It's up to meeting chairs to decide whether to reschedule if few people can attend (e.g. if there are a lot of US or Canadians in the meeting).
 
+*July 4, 2020 (the legal public holiday for US Independence Day), falls on a Saturday. For staff, Friday, July 3, may be treated as a holiday.
```
Comment (amyblais) [inline source/process/working-at-mattermost.rst]:
```suggestion
+July 4, 2020 (the legal public holiday for US Independence Day), falls on a Saturday. For staff, Friday, July 3, may be treated as a holiday.
```

File: source/process/working-at-mattermost.rst [hunk 4]
```diff
@@ -118,27 +118,30 @@ Please take off holidays relevant to your culture, region/country and preference
...
+
++December 26, 2020 falls on a Saturday.
```
Comment (amyblais) [inline source/process/working-at-mattermost.rst]:
```suggestion
++December 26, 2020 falls on a Saturday.
```

**Pull request merged**